integration-tests/ci-visibility/test-optimization-wrong-init/vitest-sum-wrong-init-test.mjs

```diff
@@ -0,0 +1,13 @@
+import tracer from 'dd-trace'
+import { describe, it, expect } from 'vitest'
+
+tracer.trace('sum.test', { resource: 'sum.test.js' }, () => {
+  describe('sum', () => {
+    it('should return the sum of two numbers', async () => {
+      // we need to give time for the tracer to flush the spans
+      // otherwise the vitest worker process will exit before the spans are flushed
+      await new Promise(resolve => setTimeout(resolve, 2000))
```

Comment:
Instead of hardcoding this to 2s, can we reduce the flushInterval of the tracer?
https://docs.datadoghq.com/tracing/trace_collection/library_config/nodejs/#traces